### PR TITLE
todos list

### PR DESCRIPTION
**Pull request by Anu-500apps**
Video link - https://drive.500apps.com/3e958a2e

### CODE REVIEW

File: assignment-5/src/components/TodosList/List.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+   <div class="todo-list" >
+    <div class="todo-item" v-for="todo in todos" :key="todo.id">
+      <table>{{ todo.title }}</table>
```
Comment (sriram-500apps) [inline assignment-5/src/components/TodosList/List.vue]: use b-table

File: assignment-5/src/components/TodosList/TodosForm.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<template>
+  <form class="todo-form" @submit="submitTodo">
```
Comment (sriram-500apps) [inline assignment-5/src/components/TodosList/TodosForm.vue:2]: user bootstrap vue

File: assignment-5/src/components/TodosList/TodosForm.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import Store from '../../store'
+export default {
+  data() {
+    return {
+      todo: '',
+    };
+  },
+  computed(){},
+   mounted(){
+    console.log("store",Store)
```
Comment (sriram-500apps) [inline assignment-5/src/components/TodosList/TodosForm.vue]: ?

**Comment by sriram-500apps** [comment]
@Anu-500apps I didn't see any documentation for methods, code logic, add comments in correct way

**Comment by sriram-500apps** [comment]
Fix there:
-----
- [x] Why New task is added at the last?
- [x] Input and button is not looking good
- [x] Input length should be big. I can't the complete text entered
- [x] completed task count should not be at bottom
- [ ] Today tasks should be a accordion and text should be at right side of the accordion
- [x] No proper gap between each task

File: assignment-5/src/App.vue [hunk 4]
```diff
@@ -1,25 +1,28 @@
 <template>
```
Comment (sriram-500apps) [inline assignment-5/src/App.vue:1]: do router setup properly

**Review by ghost — APPROVED** [review]

File: assignment-5/src/store.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
+import Vue from "vue";
```
Comment (sriram-500apps) [inline assignment-5/src/store.js:1]: this should be named as index.js and kept in store folder like we doing for router

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**